src/OrchardCore.Modules/OrchardCore.Tenants/Views/Admin/Edit.cshtml

```diff
@@ -80,7 +80,7 @@
                     </select>
                 </div>
             }
-            <div class="mb-3 col-md-6" asp-validation-class-for="TablePrefix">
+            <div class="mb-3 col-md-6 tablePrefix" asp-validation-class-for="TablePrefix">
```

Comment:
Okay, just saw that we need the class so that the script can hide it.

Reply:
Exactly

Reply:
So I think it's fine to merge this